src/Paket.Core/Dependencies/NuGetCache.fs

```diff
@@ -176,7 +179,8 @@ let tryGetDetailsFromCache force nugetURL (packageName:PackageName) (version:Sem
                 if (PackageName cachedObject.PackageName <> packageName) ||
                     (cachedObject.Version <> version.Normalize())
                 then
-                    traceVerbose (sprintf "Invalidating Cache '%s:%s' <> '%s:%s'" cachedObject.PackageName cachedObject.Version packageName.Name (version.Normalize()))
+                    if verbose then
```

Comment:
macht den check nicht auch traceVerbose schon?

Reply:
yes, but the right side (sprintf ...) would be evaluated even if verbose is false.
This was serious perf problem in the past

Reply:
Should we remove `traceVerbose` in that case ?